src/commonMain/kotlin/io/github/detekt/sarif4k/SarifSerializer.kt

```diff
@@ -12,7 +12,7 @@ object SarifSerializer {
 
     fun toMinifiedJson(sarif: SarifSchema210): String = Json.encodeToString(sarif)
 
-    fun toJson(sarif: SarifSchema210): String = json.encodeToString(sarif)
+    fun toJson(sarif: SarifSchema210): String = json.encodeToString(sarif) + "\n"
```

Comment:
Does encodeToString use strictly `\n` or Platform-sensitive new lines? Match behaviour.

Reply:
Looking at the tests, yes, they use `\n`. https://github.com/Kotlin/kotlinx.serialization/pull/2506/files#diff-e3bdf13d8a8ae78467a3aeecb08a9a8a4539a73ad45145a8ed9d1e43548ff6c8R64